pkg/controller/persistentvolume/persistentvolume_claim_binder_controller.go

```diff
@@ -305,16 +326,20 @@ func syncClaim(volumeIndex *persistentVolumeOrderedIndex, binderClient binderCli
 			return fmt.Errorf("Unexpected error getting persistent volume: %v\n", err)
 		}
 
+		// TODO:  Can this be removed? The ref will not be nil for volumes bound by this controller.
```

Comment:
> I would leave this in place

I think I disagree.  This code would allow a link to happen in reverse, right?  A PVC saying, "i've got that name" and then this controller retro-actively making it so.  Why would I ever want both paths in place?